README.md

```diff
@@ -605,7 +605,7 @@ For the client package, see https://godoc.org/github.com/theupdateframework/go-t
 
 For the client CLI, see https://github.com/theupdateframework/go-tuf/tree/master/cmd/tuf-client.
 
-## Development
+## Contributing and Development
 
 For local development, `go-tuf` requires Go version 1.16 or 1.17.
```

Comment:
This implies we can't use latest Go 1.18.x, which I don't think is true? Should we update this?

Reply:
Let's fix in https://github.com/theupdateframework/go-tuf/issues/307